cap-ci/pipeline.template

```diff
@@ -406,6 +610,36 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $test }}
+  {{- if $config.create_k8s }}
+  on_failure:
+    do:
+      - task: delete-k8s
+        privileged: true
+        timeout: 2h30m
+        input_mapping:
+          pool.kube-hosts: {{$Backend}}-pool.kube-hosts
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: splatform/catapult
+          inputs:
+          - name: catapult
+          - name: pool.kube-hosts
+          params:
+            BACKEND: {{$Backend}}
+            QUIET_OUTPUT: true
+            DEFAULT_STACK: cflinuxfs3
```

Comment:
Same here, missing `DOWNLOAD_CATAPULT_DEPS: false`.

This job should be templated and not repeated, or better, have only 1 job in a reconciler pipeline.

Reply:
same as above